tests/rptest/services/redpanda.py

```diff
@@ -3432,12 +3432,6 @@ def write_node_conf_file(self,
         except:
             cur_ver = None
 
-        # This node property isn't available on versions of RP older than 23.2.
-        if cur_ver and cur_ver >= (23, 2, 0):
-            memory_allocation_warning_threshold_bytes = 256 * 1024  # 256 KiB
```

Comment:
I was originally scraping all the test results to find oversized allocations. And to avoid too much log spam I set it to 512K initially, then lowered it to 256K, with the eventual goal of lowering it to 128K once the larger oversized allocations were fixed.